app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -783,15 +788,17 @@ private void closeTab(int index) {
           updateTabSwitcherIcon();
         })
         .show();
-    openDefaultTab();
-    updateTabSwitcherIcon();
+    if (webViewList.size() == 0) {
+      openDefaultTab();
+    } else {
+      updateTabSwitcherIcon();
+    }
   }
 
   private void openDefaultTab() {
     new Handler().postDelayed(() -> {
-      if (webViewList.size() == 0) {
         newTab(HOME_URL);
-      }
+        hideTabSwitcher();
     }, 100);
   }
```

Comment:
Can you add a blank line after this line so that there is a blank line after the method definition is complete?

Reply:
Okay. I have made the change.